### ISSUE TITLE
Start with collapsed panel

### ISSUE THREAD

**Issue by gkvoelkl** [terrestris/react-geo#1102]
I am using

```
 <Panel
        collapsible={true} 
```

The panel starts uncollapsed. I did not find a property to start with a collapsed panel.
Is there any? Thanks.

**Comment by annarieger** [comment]
The property `collapsible` is used to decide if the panel should be collapsible or not (if not, no collapsible tool will be shown in the titlebar).

Currently panels will always be instantiated uncollapsed (s. also [here](https://github.com/terrestris/react-geo/blob/master/src/Panel/Panel/Panel.jsx#L196))

**Comment by annarieger** [comment]
After https://github.com/terrestris/react-geo/pull/1103 is merged the panel component can be rendered collapsed initially if configured like 
```
<Panel
  collapsible={true}
  collapsed={true}
  [...]
</Panel> 
```

**Comment by annarieger** [comment]
Available now with [v11.4.0](https://github.com/terrestris/react-geo/releases/tag/v11.4.0)
